requirements.txt

```diff
@@ -1,9 +1,13 @@
+bottleneck
+cartopy
+cython
+fsspec
+numcodecs
 numpy>=1.19.4
 pandas>=1.1.4
+rasterio
```

Comment:
I would go with not including rasterio and cartopy in the requirements, and I don't think cython is needed. I know they are used in the generate_test_plots script, but that is only for github actions really, and as you found out, cartopy, rasterio, and satpy don't all work well together and are a bit hard to get working in a single environment.